test/utils/tools/test_result.ts

```
@@ -0,0 +1,116 @@
+    assert.equal(isUserTextResponse(r), false);
+  });
+});
+
+describe("extractImageData", () => {
+  it("returns the imageData string when present", () => {
+    const r = makeResult({ data: { imageData: "BASE64..." } });
+    assert.equal(extractImageData(r), "BASE64...");
+  });
+
+  it("returns undefined when imageData is missing", () => {
+    assert.equal(extractImageData(makeResult({ data: { foo: "bar" } })), undefined);
```

ESLint

[error] Replace `extractImageData(makeResult({·data:·{·foo:·"bar"·}·})),·undefined` with `⏎······extractImageData(makeResult({·data:·{·foo:·"bar"·}·})),⏎······undefined,⏎····`

(prettier/prettier)